src/test/java/com/google/api/codegen/metacode/SampleInitCodeTest.java

```diff
@@ -309,7 +328,24 @@ public void testMultipleMapEntries() throws Exception {
         Arrays.asList("key1", "key2", "stringmap", "123", "456", "intmap", "root");
 
     InitCodeNode rootNode =
-        InitCodeNode.createTree(getContextBuilder().dottedPathStrings(fieldSpecs).build());
+        InitCodeNode.createTree(getContextBuilder().initFieldConfigStrings(fieldSpecs).build());
+    List<String> actualKeyList = new ArrayList<>();
+    for (InitCodeNode node : rootNode.listInInitializationOrder()) {
+      actualKeyList.add(node.getKey());
+    }
+    Truth.assertThat(actualKeyList.equals(expectedKeyList));
+  }
+
+  @Test
+  public void testMultipleFormattedEntries() throws Exception {
+    List<String> fieldSpecs =
+        Arrays.asList(
+            "formatted_field%entity1", "formatted_field%entity2", "formatted_field%entity3");
```

Comment:
You could also add an =value for each entry, and then check that the initValueConfig created has those values.